youtube_dl/extractor/jamendo.py

```diff
@@ -0,0 +1,94 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..compat import compat_urlparse
+
+
+class JamendoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?jamendo\.com/track/(?P<id>[0-9]+)/(?P<display_id>[\w-]+)'
+    _TEST = {
+        'url': 'https://www.jamendo.com/track/196219/stories-from-emona-i',
+        'md5': '697564d8b10c60af35edda4fafb3fbfd',
+        'info_dict': {
+            'id': '196219',
+            'display_id': 'stories-from-emona-i',
+            'ext': 'mp3',
+            'title': 'Stories from Emona I',
+            'thumbnail': 're:^https?://.*\.jpg',
+            'url': 'https://mp3d.jamendo.com/download/track/196219/mp32'
+        }
+    }
+
+    def _real_extract(self, url):
+        url_data = self._VALID_URL_RE.match(url)
+        track_id = url_data.group('id')
+        webpage = self._download_webpage(url, track_id)
+
+        thumbnail = self._html_search_regex(r'<meta itemprop="image" content="(.+)">', webpage, 'thumbnail', fatal=False)
+        title = self._html_search_regex('<meta itemprop="name" content="(.+)">', webpage, 'title')
+
+        return {
+            'id': track_id,
+            'display_id': url_data.group('display_id'),
+            'ext': 'mp3',
+            'title': title,
+            'thumbnail': thumbnail,
+            'url': 'https://mp3d.jamendo.com/download/track/%s/mp32' % track_id
```

Comment:
Uh, I interpreted that as "URLs the API is using", I'm sorry. Additionally only mp3 and ogg seem to be available via that method:
```json
{
    "stream": {
        "mp3": "https://mp3l.jamendo.com/?trackid=196219&format=mp31&from=app-97dab294&u=3118107",
        "ogg": "https://ogg.jamendo.com/?trackid=196219&format=ogg1&from=app-97dab294&u=3118107"
    },
    "download": {
        "mp3": "https://mp3d.jamendo.com/download/track/196219/mp32/?u=3118107",
        "ogg": "https://ogg.jamendo.com/download/track/196219/ogg1/?u=3118107"
    }
}
```
On the upside I could easily extract them from the sites source-code and don't have to hope Jamendo doesn't change them in the future (but that could also hapen to this data-dict...).
Is that the reason you prefer them?
I personally would prefer more formats, but that decision is obviously up to you.